sdk/src/main/java/com/mapbox/maps/ViewAnnotationManagerImpl.kt

```diff
@@ -34,6 +36,9 @@ internal class ViewAnnotationManagerImpl(
   // struct needed for drawing, declare it only once
   private val currentViewsDrawnMap = HashMap<String, ScreenCoordinate>()
 
+  @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
+  internal val viewPositionUpdatedListenerList = CopyOnWriteArrayList<OnViewAnnotationPositionUpdatedListener>()
```

Comment:
nit: could be a `CopyOnWriteArraySet`

Reply:
👍 
also added a comment why copy on write is used